appservice/src/startStreamingLogs.ts

```diff
@@ -26,12 +26,12 @@ function getLogStreamId(client: SiteClient, logsPath: string): string {
     return `${client.id}${logsPath}`;
 }
 
-export async function startStreamingLogs(client: SiteClient, verifyLoggingEnabled: () => Promise<void>, logStreamLabel: string, logsPath: string = ''): Promise<ILogStream> {
+export async function startStreamingLogs(context: IActionContext, client: SiteClient, verifyLoggingEnabled: () => Promise<void>, logStreamLabel: string, logsPath: string = ''): Promise<ILogStream> {
     const logStreamId: string = getLogStreamId(client, logsPath);
     const logStream: ILogStream | undefined = logStreams.get(logStreamId);
     if (logStream && logStream.isConnected) {
         logStream.outputChannel.show();
-        void ext.ui.showWarningMessage(localize('logStreamAlreadyActive', 'The log-streaming service for "{0}" is already active.', logStreamLabel));
+        void context.ui.showWarningMessage(localize('logStreamAlreadyActive', 'The log-streaming service for "{0}" is already active.', logStreamLabel));
         return logStream;
     } else {
         await verifyLoggingEnabled();
```

Comment:
I can't add a comment on line 48, but do you not need to change the `context` name here as well?  Sort of like how you did here: https://github.com/microsoft/vscode-azuretools/pull/950/files#diff-e9047f508bb43125f06a6038716be014c3a833be7736051e70366cddf865e89fR36

Reply:
We don't strictly _need_ to, but yes it's a good idea to do that